docs/reference/glossary.rst

```diff
@@ -1300,6 +1311,15 @@ Glossary
     Ubuntu Server
         *Work in Progress*
 
+    Ubuntu SRU Team
+        *Work in Progress*
+
+        See also: `Ubuntu SRU Team <https://wiki.ubuntu.com/StableReleaseUpdates#Contacting_the_SRU_team>_
```

Comment:
**issue**: ReadTheDocs fails to build this PR with this warning:
```
/home/docs/checkouts/readthedocs.org/user_builds/canonical-ubuntu-packaging-guide/checkouts/69/docs/reference/glossary.rst:1299: WARNING: Inline interpreted text or phrase reference start-string without end-string. [docutils]
```

I think the problem is this:
```suggestion
        See also: `Ubuntu SRU Team <https://wiki.ubuntu.com/StableReleaseUpdates#Contacting_the_SRU_team>`_
```

Reply:
RTD build succeeded now with that patch 🎉